### ISSUE TITLE
Review Dialect: Ukrainian (Python) 

### ISSUE THREAD

**Issue by madiedgar** [legesher/legesher-translations#36]
## Review Ukrainian Language Translations for the Python Programming Language

Version 1 of Legesher only includes translations for the Python programming language. In order to add this language to the project, complete the steps detailed in [the translation guide](https://legesher.readme.io/docs/translation-guide). 

**Reviewing** a translation is extremely important as we want to best understand the culture and context behind the words we select as keywords for the programming languages. The more perspectives the better! 👀 

>⚠️ Note: this issue is to review the Python keywords and builtin functions only. Any additional translation reviews are encouraged, but not necessary. 

**To complete this issue**:
- [ ] Navigate to the _Python_ section of the dialect's `locale/DIALECT.yml` file
- [ ] Go through each of the translations for the Python keywords and built in functions adding commentary as comments
  - Leave a `✅`after a translation if you agree that's the right word to give context to the keyword
  - Leave a `❓` after a translation if you think there may be a better translation / form of the translation
  - Leave a  `🤔` after a word that you're not sure if there is a translation
- [ ] Create a pull request with changes into branch titled `ukrainian-translation` on this repo
- [ ] In your pull request, ask for a review from the original translators in the _Reviewers_ side bar section

### Reviewers Needed
In order to add the translations into our database to use, the translations will need to be approved and accepted by 3 additional native / fluent speakers. If you would like to be one of the reviewers for this language, please mention in this issue! :smile: Also feel free to loop in a friend that might be able to help with the translations!

The translated word needs to have `# ✅✅✅` in the comments after it in order to be accepted. Please take your time in being thorough with these translations. Think back when you were learning how to program and how context of words help you bridge together meaning and functionality.

**Comment by cos1715** [comment]
I can take a look

**Comment by madiedgar** [comment]
That would be amazing @cos1715!

**Comment by cos1715** [comment]
Created PR 
pls check

**Comment by valdeadmario** [comment]
Can i take this?

**Comment by madiedgar** [comment]
Absolutely @valdeadmario! Would love your help on this 👍

**Comment by madiedgar** [comment]
Hey @cos1715, @jekRock, and @Jam-Iko 👋 , thanks so much for helping with adding the initial Gujarati translation. Do you know anyone else who might be interested in contributing to this? Thank you 👍 

Hey @valdeadmario 👋 , are you still interested in helping with this issue. Would be a great help! 👍

**Comment by JekRock** [comment]
Hi @madiedgar !
I can review the translation if there have been any changes since the last review.

**Comment by Jam-Iko** [comment]
Hi @madiedgar!
I have shared this opportunity with local community, hope this will help bring in volunteers )))

**Comment by hidalgo-vntu** [comment]
I can take a look

**Comment by madiedgar** [comment]
That would be very helpful @hidalgo-vntu! Let me know if you have any questions 👍

**Comment by milafreckled** [comment]
Hi, is it available for a review? I would like to contribute, Ukrainian is my native language.

**Comment by madiedgar** [comment]
Yes @milafreckled that would be great! I'll assign you. Let me know if you have any questions.

**Comment by milafreckled** [comment]
Thank you! I've just finished reviewing in my forked repository what should I do next?
